release/create_release_archives.sh

```diff
@@ -76,7 +76,7 @@ function create_windows_archive() {
 
 pushd ${ROOT}
 cp istio.VERSION "${COMMON_FILES_DIR}"/
-find samples install -type f -name "*.yaml" \
+find samples install -type f -name "*.yaml" -o -name "cleanup*" \
```

Comment:
yes, there is currently only one cleanup script but wanted to leave it open for any others that may be added